initfiles/sbin/hpcc_setenv.in

```diff
@@ -55,6 +70,11 @@ function pidwait_fn () {
 
     if [[ $TIMEOUT -le 0 ]] && ps -p $WATCH_PID -o command= | grep "${PROCESS}" &>/dev/null ; then
       log "Failed to kill ${PROCESS} with SIGTERM"
+      if [[ $FORCE -eq 1 ]]; then
+        kill -SIGKILL $WATCH_PID > /dev/null 2>&1
+        log "killing ${PROCESS} (pid: $WATCH_PID) with SIGKILL"
```

Comment:
Should we remove $PID file here before returning ?  As it does below -

rm -f $PID > /dev/null 2>&1